core/field.js

```diff
@@ -547,6 +554,31 @@ Blockly.Field.prototype.getDisplayText_ = function() {
 };
 
 /**
+ * Get the text from this field as displayed on screen.  May differ from getText
```

Comment:
This also looks like a recurring issue. We currently have this already defined twice as well.